<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tabPlayers">
      <attribute name="title">
       <string>Players</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxPlayerOne">
         <property name="title">
          <string>Black Player</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelPlayerName">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditBlackName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelP2Species">
            <property name="text">
             <string>Species</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxBlackSpecies">
            <property name="currentText">
             <string>Computer</string>
            </property>
            <item>
             <property name="text">
              <string>Computer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Human</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Network</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelP1Url">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Url</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditBlackUrl">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelHandicap">
            <property name="text">
             <string>Handicap</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinBoxHandicap">
            <property name="maximum">
             <number>41</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBoxFreePlacement">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelFreePlacement">
            <property name="text">
             <string>Free</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxPlayerTwo">
         <property name="title">
          <string>White Player</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelWhiteName">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditWhiteName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelWhiteSpecies">
            <property name="text">
             <string>Species</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxWhiteSpecies">
            <item>
             <property name="text">
              <string>Computer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Human</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Network</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelWhiteUrl">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Url</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditWhiteUrl">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelKomi">
            <property name="text">
             <string>Komi</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxKomi">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>9.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>6.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGoGoGo">
      <attribute name="title">
       <string>GoGoGo</string>
      </attribute>
      <widget class="QLabel" name="labelBoardSize">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>103</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Board Size</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxBoardSize">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>128</width>
         <height>38</height>
        </rect>
       </property>
       <property name="currentText">
        <string>9x9</string>
       </property>
       <item>
        <property name="text">
         <string>9x9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13x13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19x19</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tabAppearance">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabEngine">
      <attribute name="title">
       <string>Engine</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Command</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditEnginePath">
         <property name="text">
          <string>/usr/games/gnugo</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
